src/testUtils/testDefaults.ts

```diff
@@ -37,30 +39,30 @@ export const DEFAULT_REQUEST_ID = 'ARequestID'
 export const INVALID_ENTITY_ID = 'invalid/entity/id' as unknown as number
 
 export const DEFAULT_DUE_DATE = {
-    recurring: false,
+    isRecurring: false,
     string: 'a date string',
     date: DEFAULT_DATE,
 }
 
 export const INVALID_DUE_DATE = {
     ...DEFAULT_DUE_DATE,
-    recurring: 'false',
+    isRecurring: 'false',
 }
 
 export const DEFAULT_QUICK_ADD_RESPONSE: QuickAddTaskResponse = {
```

Comment:
In general these responses should be updated to v9, see also some relevant comments below.